data/human/weapons.txt

```diff
@@ -333,6 +333,91 @@ effect "heavy laser impact"
 	"random velocity" 1
 	"velocity scale" -.007
 
+outfit "Mining Laser"
+	category "Guns"
+	cost 38000
+	thumbnail "outfit/mining laser"
+	"mass" 12
+	"outfit space" -12
+	"weapon capacity" -12
+	"gun ports" -1
+	weapon
+		sprite "projectile/mining laser"
+			"frame rate" 1
+		sound "mining laser"
+		"hit effect" "beam laser impact"
+		"inaccuracy" .35
+		"velocity" 200
+		"lifetime" 1
+		"reload" 1
+		"firing energy" 1.1
+		"firing heat" 2.1
+		"shield damage" .8
+		"hull damage" 1.4
+		"minable damage" 1.9
+		"prospecting" 1
+	description "The development of the Beam Laser in the mid-2400s not only led to a revolution in space warfare, but also to the field of asteroid mining, as the traditional but inefficient method of towing asteroids to bases for extraction was supplanted by individual spacecraft that could precisely extract the ore from asteroids themselves. The modern Mining Laser is a derivative of the Beam Laser engineered to optimize mineral yield and extraction rates at the expense of range and shield damage."
```

Comment:
Let's break the lore across both mining weapons. This allows us to keep the lengths down as well.
```suggestion
	description "The development of the Beam Laser in the mid-2400s not only led to a revolution in space warfare, but also to the field of asteroid mining, as the traditional but inefficient method of towing asteroids to bases for extraction was supplanted by individual spacecraft that could precisely extract the ore from asteroids themselves."
```

Reply:
I think we should have something alluding to the Mining Laser's prospecting attribute, which this would remove.

Reply:
Yeah. If we're to mention it in either, it probably does make sense for it to be in the gun's description since it's more common and located wherever the turret is found.

Reply:
```suggestion
	description "The development of the Beam Laser in the mid-2400s not only led to advancements in space warfare, but also to the field of asteroid mining, as the traditional but inefficient method of towing asteroids to bases for extraction was supplanted by individual spacecraft that could precisely extract the ore from asteroids themselves. The modern Mining Laser is a derivative of the Beam Laser engineered to optimize mineral yield and extraction rates at the expense of range and shield damage."
```

Reply:
```suggestion
	description "The development of the Beam Laser in the mid-2400s not only led to advancements in space warfare, but also in the field of asteroid mining, as the traditional but inefficient method of towing asteroids to bases for extraction was supplanted by individual spacecraft that could precisely extract the ore from asteroids themselves."
	description "	The modern Mining Laser is a derivative of the Beam Laser engineered to optimize mineral yield and extraction rates at the expense of range and shield damage."
```
I think "advancements in the field of asteroid mining" makes more sense than "advancements to the field of asteroid mining". At the very least it will be consistent.